core/src/main/scala/akka/persistence/dynamodb/internal/QueryDao.scala

```diff
@@ -277,7 +317,8 @@ import software.amazon.awssdk.services.dynamodb.model.QueryResponse
               writeTimestamp = getTimestamp(item),
               readTimestamp = InstantFactory.now(),
               payload = None, // lazy loaded for backtracking
-              serId = item.get(EventSerId).n().toInt,
+              serId =
+                if (item.containsKey(EventSerId)) item.get(EventSerId).n().toInt else backtrackingBreadcrumbSerId(),
```

Comment:
So I think we could just do this change. Remove the special serializer id and comments.

```suggestion
              serId = 0, // lazy loaded for backtracking (or event is in fallback storage)
```

Reply:
Forgot about the mentioned special filtered serializer id check already. So best to keep the event ser id, but don't see why it couldn't just be 0 otherwise.